### ISSUE TITLE
VPC plots stratified by more than two variables

### ISSUE THREAD

**Issue by JoannaPeng** [UUPharmacometrics/xpose#227]
How to create VPC plots stratified by more than two variables?

Specifically, the VPC was created using PsN with -stratify_on=VAR1,VAR2,VAR3,VAR4.

In R, vpc_data=vpc_data(..., stratify=VAR1~VAR3), but this line specifies only two variables. How to modify this line to specify all four variables? (And the combination of VAR3 and VAR4 will be used as a column variable, and the combination of VAR1 and VAR2 will be used as a row variable.)

I tried stratify=c('VAR1','VAR2','VAR3','VAR4'), but got an error message.

**Comment by bguiastr** [comment]
Hello @JoannaPeng, 

I have provided some answer to you about this in a different thread [here](https://github.com/UUPharmacometrics/xpose/issues/110#issuecomment-2132385955).

TL;DR: an easy solution is to generate a unique variable in your dataset that will be used for VPC stratification.

I hope this helps !
